app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -452,11 +448,9 @@ public void removeCoin(@Nullable Coin coin, Boolean collected) {
             String default_score = getString(R.string.map_score_text, localPlayer.getScore());
             currentScoreView.setText(default_score);
             //TODO: Inform database
```

Comment:
I know this is not your TODO, but you could remove it when removing the comments if possible